euth/ideas/templates/euth_ideas/idea_detail.html

```diff
@@ -10,7 +10,7 @@
                     <a href="{% url 'project-detail' idea.project.slug %}"><i class="fa fa-arrow-left" aria-hidden="true"></i> {% trans 'back to project' %}</a>
                 </li>
             </ul>
-            {% if request.user == idea.creator or request.user.is_superuser %}
+            {% if request.user == idea.creator and view.crud_enabled or request.user.is_superuser %}
```

Comment:
What about using [django-rules](https://github.com/dfunckt/django-rules#rules-and-permissions-in-templates) instead?